TrackletAlgorithm/MatchProcessor.h

```diff
@@ -349,34 +918,77 @@ void MatchCalculator(BXType bx,
   const auto kPhi0_shift         = (LAYER < TF::L4)? 3 : 0;                                      // phi0shift_ in emulation defined in MC
   const auto kShift_phi0bit      = 1;                                                             // phi0bitshift in emulation defined in constants
   const ap_uint<10> kPhi_corr_shift_L123 = 7 + kNbitsdrinv + kShift_phi0bit - kShift_Rinv - kShift_Phider;                    // icorrshift for L123
-  const ap_uint<10> kPhi_corr_shift_L456 = kPhi_corr_shift_L123 - 10 + kNbitsrL456;                                           // icorrshift for L456
-  const auto kPhi_corr_shift     = (LAYER < TF::L4)? kPhi_corr_shift_L123 : kPhi_corr_shift_L456;                            // icorrshift_ in emulation
+  const ap_uint<10> kPhi_corr_shift_L456 = kPhi_corr_shift_L123 - 10 + kNbitsrL456;
+  const auto kPhi_corr_shift     = (LAYER < TF::D1) ? ((LAYER < TF::L4)? kPhi_corr_shift_L123 : kPhi_corr_shift_L456) : ap_uint<10>(6);                            // icorrshift_/shifttmp  in emulation
```

Comment:
I find this really hard to read. Maybe adding a parenthesis for the inner tertiary operator will make it easier to read?

Reply:
I've cleaned it up to
```c++
  const auto kPhi_corr_shift     = (LAYER < TF::D1) ?
                                   ((LAYER < TF::L4)? kPhi_corr_shift_L123 : kPhi_corr_shift_L456) // barrel
                                   : ap_uint<10>(6); // disk                        // icorrshift_/shifttmp  in emulation
```